DOC-1602: Added documentation for the new model files/options (#2304)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/model.adoc

[[model]]
== `+model+`

> This option allows you to specify the model that {productname} should use. The default model included with {productname} is *dom*. Custom models are not supported at this time.

Reviewer: `you` should be avoided... a lot of old docs use it still but new docs shouldn't. See https://www.tiny.cloud/docs/configure/contributing-docs/#pronouns

```suggestion
This option allows configuring which model {productname} should use. The default model included with {productname} is *dom*. Custom models are not supported at this time.
```

Author: ok, this whole thing is 90% the `theme.adoc` file with `theme` replaced by `model`.

Reviewer: Yeah I figured that was probably the case.

The name of the model should match the name of the folder within the models directory of {productname}. If the specified model is not found, {productname} will not load.

Type : `+String+`

> Default Value : `dom`

Reviewer: ```suggestion
Default Value : `'dom'`
```
